### DISCUSSION TITLE
偶现PAGFile.LoadFromPath() Invalid pag file : /storage/emulated/0/Android/data/***r/files/pags/***.pag

### DISCUSSION THREAD

**Discussion by Mr-Zhaozhi** [Tencent/libpag#2208] [Q&A]
平台：Android
SDK版本：implementation "com.tencent.tav:libpag:4.0.5.5"
问题：lottie替换PAG，把lottie的json文件转成pag播放。
文件存放项目assets目录下的pag文件，用户加载的时候copy到沙箱/storage/emulated/0/Android/data/***r/files/pags这个目录下。偶现播放几次后报这个错误之后一直不会再好，只有把文件删了 重新配 就又好了。好了之后偶现还会出现 。求解
`E  [SurfaceView[com]#21(BLAST Consumer)21](id:74500000018,api:0,p:-1,c:1861) 
disconnect: not connected (req=1)2024-03-19 11:02:48.310  1861-1861  tgfx 
  E  PAGFile.LoadFromPath() Invalid pag file : /storage/emulated/0/Android/data/com/files/pags/laun.pag
2024-03-19 11:04:39.188  1861-1861  tgfx   
 E  PAGFile.LoadFromPath() Invalid pag file : /storage/emulated/0/Android/data/com/files/pag/laun.pag`

**Comment by kevingpqi123** [comment]
这个问题建议自己排查下，其它业务暂未反馈过类似问题，简单的方式，你添加一些 log， 通过系统的方式读取文件，看文件内容是否为空

**Comment by CodeJhF** [comment]
这条我们在当时的回复里已经给出结论，这里就不重复贴了，先关闭本条。
如果按当时的结论排查后仍有问题，欢迎回复补充最小复现 demo 与版本号（4.4 / 4.5），我们会重新看。（stale-close）
